js/apps/admin-ui/package.json

```diff
@@ -126,16 +117,15 @@
     "@types/react": "^18.3.18",
     "@types/react-dom": "^18.3.5",
     "@vitejs/plugin-react-swc": "^3.8.0",
-    "cypress": "^14.1.0",
-    "cypress-axe": "^1.6.0",
-    "cypress-split": "^1.24.14",
     "jsdom": "^26.0.0",
     "ldap-server-mock": "^6.0.1",
```

Comment:
This dependency is no longer being used and can be removed.

Reply:
right, that's what I did

Reply:
I mean that `ldap-server-mock` can be removed.

Reply:
ahh, right